test/nn/test_embedding.py

```diff
@@ -1170,6 +1170,8 @@ def _test_EmbeddingBag(
     @dtypesIfCUDA(*itertools.product((torch.int, torch.long), (torch.int, torch.long),
                                      (torch.float, torch.double, torch.half)))
     def test_embedding_bag_device(self, device, dtypes):
+        if torch.bfloat16 in dtypes and device=="cpu":
+            self.skipTest("bfloat16 not supported with Jetson cpu")
```

Comment:
Err? This would disable the test on all CPU, not just on jetson?

Reply:
sorry typo from recent commit